armv7m7-imxrt106x/low.c

```diff
@@ -218,29 +218,26 @@ void low_copyfrom(u16 segm, u16 offs, void *dst, unsigned int l)
 void low_memcpy(void *dst, const void *src, unsigned int l)
 {
 	asm volatile(" \
-		mov r1, %2; \
-		mov r3, %1; \
-		mov r4, %0; \
-		orr r2, r3, r4; \
-		ands r2, #3; \
+		orr r3, %0, %1; \
+		ands r3, #3; \
 		bne 2f; \
 	1: \
-		cmp r1, #4; \
+		cmp %2, #4; \
 		ittt hs; \
-		ldrhs r2, [r3], #4; \
-		strhs r2, [r4], #4; \
-		subshs r1, #4; \
+		ldrhs r3, [%1], #4; \
+		strhs r3, [%0], #4; \
+		subshs %2, #4; \
 		bhs 1b; \
 	2: \
-		cmp r1, #0; \
+		cmp %2, #0; \
 		ittt ne; \
-		ldrbne r2, [r3], #1; \
-		strbne r2, [r4], #1; \
-		subsne r1, #1; \
+		ldrbne r3, [%1], #1; \
+		strbne r3, [%0], #1; \
+		subsne %2, #1; \
 		bne 2b"
 	:
 	: "r" (dst), "r" (src), "r" (l)
```

Comment:
We might need lo register to be used. Perhaps force them to be used by specifying:
`"l" (dst), "l" (src), "l" (l)`?